test/Microsoft.ApplicationInsights.AspNetCore.Tests/JavaScript/ApplicationInsightsJavaScriptTest.cs

```diff
@@ -11,19 +13,21 @@
 
     public static class ApplicationInsightsJavaScriptTest
     {
+        private static JavaScriptEncoder encoder = JavaScriptEncoder.Create(new UnicodeRange[] { UnicodeRanges.BasicLatin });
```

Comment:
Yes that is how the documentation shows it being used.
Off the top of my head I don't know what the framework defaults that configuration to but I can look that up via debugging.  For the unit test this is all that is needed.